pkg/server/router.go

```diff
@@ -48,6 +48,9 @@ func router(ctx context.Context, config *Config, cfg *cmds.Server) http.Handler
 	authed.Path(prefix + "/server-ca.crt").Handler(fileHandler(serverConfig.Runtime.ServerCA))
 	authed.Path(prefix + "/config").Handler(configHandler(serverConfig, cfg))
 	authed.Path(prefix + "/readyz").Handler(readyzHandler(serverConfig))
+	authed.Path(prefix + "/encrypt/status").Handler(encryptionStatusHandler(serverConfig))
+	authed.Path(prefix + "/encrypt/enable").Handler(encryptionEnableHandler(serverConfig))
+	authed.Path(prefix + "/encrypt/stage").Handler(encryptionStageHandler(serverConfig))
```

Comment:
I'd still like to see these merged into a single `encrypt/config` handler. Having an `enable` handler that takes a struct with only a single `enable` field feels not very idiomatic. Can we merge these together and move the `enable` field into the `EncryptionRequest` struct?
```suggestion
	authed.Path(prefix + "/encrypt/enable").Handler(encryptionEnableHandler(serverConfig))
	authed.Path(prefix + "/encrypt/stage").Handler(encryptionStageHandler(serverConfig))
```

Reply:
Reworked into single handler for both enable and stages